### PR TITLE
Bump Go versions

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 
 go:
-  - 1.5
-  - 1.6
-  - 1.7
+  - 1.5.x
+  - 1.6.x
+  - 1.7.x
+  - 1.8.x
+  - tip
```
Comment (sbinet) [inline .travis.yml]: `tip` is deprecated.
one should use `master` instead.

**Comment by gdamore** [comment]
I'm actually thinking of updating this to remove even the older releases.  Technically golang folks only support 1.7 and 1.8 now... and there are a few things I'd like to revisit again that I couldn't do because of missing support in earlier versions of golang.

@sbinet do you have a reference about tip vs master?

**Comment by sbinet** [comment]
yep:
https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use

**Comment by gdamore** [comment]
This was addressed in #307

**Pull request closed without merging**